content/en/internal/configuration-database/access/_index.md

```diff
@@ -1,64 +1,65 @@
 ---
 title: "Access"
-description: "Accessing GOCDB"
-weight: 20
+description: "Accessing the Configuration Database"
+weight: 25
 type: "docs"
 ---
 
-The GOCDB can be accessed at [https://goc.egi.eu](https://goc.egi.eu).
+To access [web interface](https://goc.egi.eu) of the EGI Configuration Database
```

Comment:
```suggestion
To access the [web interface](https://goc.egi.eu) of the EGI Configuration Database
```

Reply:
It seems this point is one of or even the latest one to be addressed before being able to merge, no?